### ISSUE TITLE
After install docker-desktop all my containers and images disappeared

### ISSUE THREAD

**Issue by Rombersoft** [docker/desktop-linux#8]
Yesterday at morning I had 5 images and 3 containers. At evening I installed at first docker-desktop. Today I adjusted pass in order to log in and saw "No containers running". I've switched to CLI, typed docker images and docker ps -a and both commands got empty result.
I've checked /var/lib/docker directory and it size was 4.4G. So I understand that images and containers exist in /var/lib/docker directory but is it dead?

**Comment by p1-0tr** [comment]
Hi, I am not a 100% sure that I understand correctly. I assume you were running a regular docker engine (e.g. `docker-ce`) and then installed docker desktop. If so you can either, switch your CLI context to `default` - `docker context use default` (https://docs.docker.com/engine/context/working-with-contexts/) - or shut down docker desktop, which in turn will restore your CLI context, like that you will be able to see your images running in the regular engine. Docker Desktop comes with its own engine, which runs in a VM (for more info see - https://docs.docker.com/desktop/linux/#why-docker-desktop-for-linux-runs-a-vm), so the images, containers, etc. will be separate from what you would run in a natively installed engine (e.g. `docker-ce`).

**Comment by Rombersoft** [comment]
shuting down docker desktop restored my CLI context. Oh Alah!
But how can I import images and containers to desktop-linux context?

**Comment by aamirglb** [comment]
Hello, 
Even I faced the same issue. After installing and starting DD4L, I was not able to see my existing images/containers using   `docker images` and `docker container ls`. It would be nice if we can import our existing images/containers into DD4L instead of pulling the images again into DD4L and starting fresh containers in DD4L.

**Comment by souravjamwal77** [comment]
Yes, I am also facing the same problem. After installing Docker Desktop, I want to be able to import my existing images/containers from docker engine/cli to desktop

**Comment by aditya-kapre-shrewsbury** [comment]
> Hi, I am not a 100% sure that I understand correctly. I assume you were running a regular docker engine (e.g. `docker-ce`) and then installed docker desktop. If so you can either, switch your CLI context to `default` - `docker context use default` (https://docs.docker.com/engine/context/working-with-contexts/) - or shut down docker desktop, which in turn will restore your CLI context, like that you will be able to see your images running in the regular engine. Docker Desktop comes with its own engine, which runs in a VM (for more info see - https://docs.docker.com/desktop/linux/#why-docker-desktop-for-linux-runs-a-vm), so the images, containers, etc. will be separate from what you would run in a natively installed engine (e.g. `docker-ce`).

What is the context changed to after installing/starting desktop ?

**Comment by arvin1408** [comment]
Issue still persists. On Ubuntu 22.04, starting Docker Desktop removes context of previous local builds and only shows the ones from Docker Desktop registry.

**Comment by rodrigoGA** [comment]
same problem with ubntu 22.04.
I had volumes with important information, and development environments with configurations that were difficult to replicate.
When installing docker-desktop everything is gone, any idea how to get it back?

**Comment by aamirglb** [comment]
Please check @aditya-kapre-shrewsbury comment above on how to restore your old docker volumes. You need to shut down docker desktop to restore the old images.

**Comment by charelstoncrabb** [comment]
Also had a minor panic attack after installing DD4L and it looked like all my conatiners, images and volumes were gone! Is there any update on and/or plan to be able to move existing docker stuffs into the docker desktop context? I'd like to be able to use docker desktop and with how much stuff I'm running/is living in volumes, its just inconvenient enough to not move it over manually--provided there is even a good way to do this manually without accidentally nuking everything which is what I originally thought happened.

**Comment by HammadRafique29** [comment]
Don't worry guys. Do this to use your old docker context in docker desktop:

- Open Docker Settings
- Goto Builders
- You can see at least 2 docker builders
- Click 3-dots next to "default" (my case)
- Click Use.

Now you can use your old docker context. Thanks
Also let's connect before AI replaces us 💯 

![Image](https://github.com/user-attachments/assets/c3e4bc9f-3184-479e-8e4e-344b3a3653ff)
